onnxruntime_extensions/tools/pre_post_processing/steps/vision.py

```diff
@@ -522,12 +522,13 @@ class ImageBytesToFloat(Step):
     Convert uint8 or float values in range 0..255 to floating point values in range 0..1
     """
 
-    def __init__(self, name: Optional[str] = None):
+    def __init__(self, rescale_factor: float = 1/255, name: Optional[str] = None):
```

Comment:
If we insert a new arg in the middle won't that break existing usage where the name is being specified?

Reply:
We don't have a reference in the repo, so we wouldn't break it.
we may break the case of the other Users' code.